apps/crm/src/pages/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { NextPage } from "next";
+import React, { useEffect } from "react";
+import { useAppBridge } from "@saleor/app-sdk/app-bridge";
+
+import { useRouter } from "next/router";
+
+const IndexPage: NextPage = () => {
+  const { appBridgeState, appBridge } = useAppBridge();
+  const { replace } = useRouter();
+
+  useEffect(() => {
+    if (appBridgeState?.ready) {
+      replace("/configuration/providers/Mailchimp");
```

Comment:
I know changing it would be quite a chore, but I feel like we should use lowercase names for providers across the app. In the config, we can have a separate field for the display name (`label` or something), but we should probably default to lowercase everywhere else.